src/common.ts

```diff
@@ -0,0 +1,8 @@
+export enum Value {
+  Scissors = "Scissors",
+  Paper = "Paper",
+  Rock = "Rock",
+  Spock = "Spock",
+  Lizard = "Lizard",
+  Null = "",
```

Comment:
Tutaj chyba poszedłbym bardziej w 'Empty' - bo null i pusty string to ciut inne rzeczy